drydock/templates/kustomized/tutor13/extensions/ingress.yml

```diff
@@ -34,7 +34,7 @@ spec:
             port:
               number: 80
     {%- endfor %}
-  {% if ENABLE_HTTPS -%}
```

Comment:
so we're always using https now? well that has happened across the majority of manifests repos

Reply:
This is mostly so we can let the ingress handle the certificates using the other variables instead of `ENABLE_HTTPS` which is tied to other configurations in tutor.

Reply:
Sorry for not getting this right but is there any implication for the current installations?

Reply:
The only one is that if you want to run the platform without https you will have to turn off two variables instead of one (`ENABLE_HTTPS` and `DRYDOCK_AUTO_TLS`, or `ENABLE_HTTPS` and `DRYDOCK_CUSTOM_CERTS`).

Reply:
good, thanks